docs/src/main/asciidoc/_cloud/Getting_Started.adoc

```diff
@@ -0,0 +1,156 @@
+
+Once you have downloaded and installed/unpacked the client, you can authenticate to your running sandbox OpenShift instance by using the `Copy Login Command` from the upper right menu in the sandbox user-interface (your username is displayed there.)
+
+Example:
+```
+$ oc login --token=sha256~b-3av_0Lp8wmJu0G3WXzdfds34454mkvWjRRESd_jHTA --server=https://api.sandbox.x8i5.p1.openshiftapps.com:6443
+```
+
+We'll be using the `someuser-dev` project for this example (`someuser` should be replaced with the username of the user):
+
+```
+$ oc project someuser-dev
+Already on project "someuser-dev" on server "https://api.sandbox.x8i5.p1.openshiftapps.com:6443"
+```
+
+In the OpenShift sandbox, it is not possible to create a new project but, in case you are running your own OpenShift, the command to create a new project is:
+```
+oc new-project <project name>
+```
+
+### Helm Charts
+https://helm.sh/[Helm] provides an easy way to manage and share applications on Kubernetes and OpenShift. We will use it to deploy our example applications to OpenShift.
+
+##### Install Helm and Helm Charts
+See the instructions for installing Helm https://helm.sh/docs/intro/install/[here]. Helm is installed on your local system as a command, which we will use to install the WildFly Charts. Once Helm is installed, we can proceed to use it to install the Helm Charts in our OpenShift instance:
+
+```
+$ helm repo add wildfly https://docs.wildfly.org/wildfly-charts/
+```
+
+For more information on WildFly Helm Charts, see this https://www.wildfly.org/news/2021/05/05/helm-charts-for-wildfly/[blog post] and the https://docs.wildfly.org/wildfly-charts/[WildFly Helm Charts documentation]
+
+## Example: To-do Quickstart
+The first example we will build and deploy to OpenShift is the https://github.com/wildfly/quickstart/tree/main/todo-backend[todo-backend quickstart]. This quickstart provides https://github.com/wildfly/quickstart/tree/main/todo-backend#run-the-backend-on-openshift[documentation] for installing via Helm, so only a brief version is detailed below.
+
+First, we need to provision a database instance, in this case PostgresSQL:
+```
+$ oc new-app postgresql-ephemeral \
+   -p DATABASE_SERVICE_NAME=todos-db \
+   -p POSTGRESQL_DATABASE=todos
+
+```
+
+Next we use the WildFly Helm Charts to build and deploy our quickstart:
+```
+$ helm install todo-backend --set build.ref={WildFlyQuickStartRepoTag} -f https://raw.githubusercontent.com/wildfly/wildfly-charts/main/examples/todo-backend/todo-backend-bootable-jar.yaml wildfly/wildfly
+```
+
+build.ref is used to indicate which quickstart tag is to be used, while build.s2i.version is set to indicate the version of the image to be pulled and used from quay.io (this is using the wildfly-centos7 and wildfly-runtime-centos7 images.)
```

Comment:
@jmfinelli , you are using bootable JAR, although there you mention s2i. Bootable JAR doesn't use wildfly-centos nor wildfly-runtime-centos7 images. The Openjdk S2I builder image is used to build and run the application.

Reply:
Thanks @jfdenise, I'm not even sure why that `build.s2i` was there.